ds3-autogen-api/src/main/java/com/spectralogic/ds3autogen/api/models/Arguments.java

```diff
@@ -16,8 +16,8 @@
 package com.spectralogic.ds3autogen.api.models;
 
 public class Arguments {
-    final private String type;
```

Comment:
Because NetNullableVariable extends Arguments, and I wanted access to the variables from the parent.